scripts/images/raspberry-pi-hypriot/scripts/build.sh

```diff
@@ -50,9 +49,8 @@ echo "RancherOS: root partition" > build/root/root.txt
 
 # unpack and cleanup the basefs
 #- doing this on a local folder keeps our resulting image clean (no dirty blocks from a delete)
+dpkg-deb -x build/kernel.deb build/basefs
 dpkg-deb -x build/bootloader.deb build/basefs
-# upgrade Raspberry Pi bootfile for RPi3 support
-tar xvzf build/rpi3-bootfiles.tar.gz -C build/basefs/boot
 # remove RPi1 kernel, we only support RPi2 and RPi3 in ARMv7 mode
 rm -fr build/basefs/boot/kernel.img
 rm -fr build/basefs/lib/modules/{4.1.17+,4.1.17-hypriotos+}
```

Comment:
Please change `4.1.17` to `4.4.27`, then the PR is fine. This removes the ARMv6 parts of the RPi kernel and reduces the size of the SD image.